.gitignore

```diff
@@ -8,3 +8,4 @@
 /tmp/
 .DS_Store
 .byebug_history
+.idea/
```

Comment:
Files and directories generated by your IDE are best ignored by a gitignore_global file, rather than adding it to repos.

```bash
$ git config --global core.excludesfile '~/.gitignore'
$ echo '.idea/' >> ~/.gitignore
```

Reply:
ok, I'll do, thank you